applications/README.md

```diff
@@ -0,0 +1,25 @@
+# Applications
+This directory contains CDK-based serverless applications.
+
+## Prerequisites
```

Comment:
Might be worth adding a section on what tools you need to run this (e.g. node, npm, cdk cli etc) or maybe link to [the docs](https://docs.aws.amazon.com/cdk/v2/guide/testing.html).

Reply:
Good shout, have added